boards/arc/hsdk/Kconfig.board

```diff
@@ -0,0 +1,11 @@
+# Kconfig - DesignWare ARC HS Development Kit board configuration
+#
+# Copyright (c) 2019 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+config BOARD_HSDK
+	bool "ARC HS Development Kit"
+	depends on SOC_ARC_HSDK
+	help
+	  The DesignWare ARC HS Development Kit is a ready-to-use platform for rapid software development on the ARC HS3x family of processors. It supports single- and multi-core ARC HS34, HS36 and HS38 processors and offers a wide range of interfaces including Ethernet, WiFi, Bluetooth, USB, SDIO, I2C, SPI, UART, I2S, ADC, PWM and GPIO. A Vivante GPU is also contained in the ARC Development System SoC. This allows developers to build and debug complex software on a comprehensive hardware platform.
```

Comment:
seems like we need some line breaks here.

Reply:
done